deployer/README.md

```diff
@@ -192,6 +196,21 @@ This allows us to optimise and parallelise the automatic deployment of our hubs.
 ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
+### `new-grafana-token`
+
+This function checks if a service account named `deployer` exists for a cluster's Grafana, and creates it if it doesn't.
+For this service account, it then generates an API token named `deployer`.
+If a token with this name already exists, it will show whether or not the token is expired
+and wait for cli input about whether to generate a new one or not.
+
+The API token is stored encrypted inside the `enc-grafana-token.secret.yaml` file of the cluster.
+If such a file doesn't already exist, it will be created by this function.
+
+  Generates:
+    - an `enc-grafana-token.secret.yaml` file if not already present
+
+  Updates:
+    - the content of `enc-grafana-token.secret.yaml` with the new token if one already existed
```

Comment:
I love these kinds of summaries!!

Could you add some clarity what kinds of API tokens (a Grafana API token?), and service accounts (a grafana service account? a k8s service account?) are meant, and also what kind of credential is used to set things up?

Reply:
Added! Thanks @consideRatio! I spent this time with my head inside Grafana-stuff and forgot there exist other things  😅